docs/2.7.0/docs/daml-finance/tutorials/getting-started/holding.rst

```diff
@@ -0,0 +1,181 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Holding
+#######
+
+This tutorial introduces the core asset model of the library through a simple example. The purpose
+is to illustrate the concepts of :ref:`account <account>`, :ref:`instrument <instrument>`, and
+:ref:`holding <holding>`, as well as showing how to work with Daml interfaces.
+
+We are going to:
+
+#. create accounts for Alice and Bob at the Bank
+#. issue a cash instrument
+#. credit a cash holding to Alice’s account
+
+We expect the reader to be familiar with the basic building blocks of Daml. If that is not the case,
+a suitable introduction can be found `here <https://www.digitalasset.com/developers/learn>`_.
+In particular, `An Introduction to Daml <https://docs.daml.com/daml/intro/0_Intro.html>`_
+would be a good starting point.
+
+This example refers to the :doc:`Transfer <transfer>` tutorial script in the sense that it explains
```

Comment:
I think the holding tutorial deservers its own script, where we just create records of ownership at the bank.

We can show
- how to create a holding using the template (the simple way)
- how this can be done via factories and why it makes sense

Reply:
I added a description of both ways to create a holding now.